.trunk/out

```diff
@@ -0,0 +1 @@
+/Users/daniel.strobl/.cache/trunk/repos/77d6c8d3281f4b5619d2de9ef4f17e81/out
```

Comment:
are you using SVN? Is this file meant to be here?

Reply:
no, no idea where this is coming from